src/Test/OpenRiaservices.EndToEnd.Wcf.Test/Data/DomainContextWithMockDomainClientTests.cs

```diff
@@ -18,6 +18,10 @@ namespace OpenRiaServices.Client.Test
     using Resources = SSmDsClient::OpenRiaServices.Client.Resources;
 
     [TestClass]
+    // TODO: Should this be moved to client.test "shared" instead ?
+#if !NETFRAMEWORK
+    [Ignore("Tests with cancellation has problems")]
```

Comment:
@SandstromErik @erikoijwall you might want to look into how to ensure this removed when enabling NET6 tests